saw-core/src/SAWCore/OpenTerm.hs

```diff
@@ -160,14 +161,14 @@ bindTCMOpenTerm m f = OpenTerm (m >>= unOpenTerm . f)
 bindPPOpenTerm :: OpenTerm -> (String -> OpenTerm) -> OpenTerm
 bindPPOpenTerm (OpenTerm m) f =
   OpenTerm $
-  do ctx <- askCtx
+  do -- ctx <- askCtx
      t <- typedVal <$> m
      -- XXX: this could use scPrettyTermInCtx (which builds in the call to
      -- PPS.render) except that it's slightly different under the covers
      -- (in its use of the "global" flag, and it isn't entirely clear what
      -- that actually does)
      unOpenTerm $ f $ PPS.render PPS.defaultOpts $
-       ppTermInCtx PPS.defaultOpts (map fst ctx) t
+       ppTermInCtx PPS.defaultOpts [] {-(map fst ctx)-} t
```

Comment:
Delete the `{-(map fst ctx)-}` comment? (And similarly below in `openTermType`.)